material.css

```diff
@@ -270,3 +270,29 @@ input.Button[type="submit"] {
     border-radius: 18px;
     margin-left: 6px;
 }
+
+.SessionDropdown .Dropdown-toggle .Avatar {
+    margin: 6px 5px -2px -6px;
+    width: 24px;
+    height: 24px;
+    border-radius: 24px;
+    font-size: 12px;
+    line-height: 24px;
+}
+
+@media (min-width: 768px)
+.NotificationsDropdown .Dropdown-toggle {
+    width: 36px;
+    text-align: center;
+}
+
+@media (min-width: 768px)
+.DiscussionPage-nav .ButtonGroup:not(.itemCount1) .Dropdown-toggle {
+    width: 22%;
+    margin-top: 9px;
+}
+
+.Button--link {
```

Comment:
Can you tell me what does this do?

Reply:
.Button--link {
    background: transparent !important;
    margin-right: 9px;
}

This fixes an alignment issue for the Top Button Links in the Header!

Reply:
Okay 👍

Reply:
This worked for the YouTube and the other button at the right but not for the notification and for the flag button, you can't see it but there is a flag button next to the notification button, can you find a fix for that?